src/Organic/SDK/OrganicSdk.php

```diff
@@ -122,19 +122,27 @@ public function getCDNUrl() {
     /**
      * Builds the SDK V1 URL to embed the JS SDK into web pages
      *
-     * @return string
+     * @return array
      */
     public function getSdkV1Url() {
-        return $this->cdnUrl . 'sdk/unit-sdk.js?' . $this->siteGuid;
+        return [
+            'default' => $this->cdnUrl . 'sdk/unit-sdk.js?' . $this->siteGuid,
+            'module' => '',
+        ];
```

Comment:
I think returning an array from `getSdkV1Url` and `getSdkV2Url` is misleading. Is there a reason we can't use a default parameter like this?
```
public function getSdkV1Url( string $type = 'default' ) {
  return $type === '' ? '' : $this->cdnUrl . 'sdk/unit-sdk.js?' . $this->siteGuid;
```